.maintain/gitlab/check_polkadot.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#
+# check if a pr is compatible with polkadot companion pr or master if not 
+# available
+#
+# mark companion pr in the body of the polkadot pr like
+#
+# polkadot companion: paritytech/polkadot#567
+
+
+github_api_polkadot_pull_url="https://api.github.com/repos/paritytech/polkadot"
+# use github api v3 in order to access the data without authentication
+github_header="Accept: application/vnd.github.v3+json" 
+
+boldprint () { printf "|\n| \033[1m${@}\033[0m\n|\n" ; }
+
+
+
+SUBSTRATE_PATH=$(pwd)
+
+# Clone the current Polkadot master branch into ./polkadot.
+git clone --depth 1 https://gitlab.parity.io/parity/polkadot.git
```

Comment:
Why does it clone from gitlab?

Reply:
I didn't change that. It probably won't be faster since the runners do not host the repo. will change to github, thanks.